src/OrbitGl/CoreMathTest.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "CoreMath.h"
+
+namespace orbit_gl {
+
+ClosedInterval kFakeClosedInterval{0, 3};
+
+TEST(ClosedInterval, FromValues) {
+  EXPECT_EQ(kFakeClosedInterval, ClosedInterval::FromValues(0, 3));
+  EXPECT_EQ(kFakeClosedInterval, ClosedInterval::FromValues(3, 0));
+}
+
+TEST(ClosedInterval, Intersects) {
+  EXPECT_TRUE((ClosedInterval{0, 3}.Intersects(ClosedInterval{1, 2})));
+  EXPECT_TRUE((ClosedInterval{1, 2}.Intersects(ClosedInterval{0, 3})));
+  EXPECT_TRUE((ClosedInterval{0, 1}.Intersects(ClosedInterval{1, 2})));
+  EXPECT_TRUE((ClosedInterval{1, 2}.Intersects(ClosedInterval{0, 1})));
+  EXPECT_FALSE((ClosedInterval{0, 1}.Intersects(ClosedInterval{2, 3})));
+  EXPECT_FALSE((ClosedInterval{2, 3}.Intersects(ClosedInterval{0, 1})));
```

Comment:
I think there are extra `( )` that are not needed

Reply:
I thought the same, but the compiler doesn't agree with us:

https://screenshot/AFodPCSuFdDJAuv

Reply:
Interesting!